src/main/java/org/team199/robot/SwerveModule.java

```diff
@@ -35,16 +34,16 @@
      *                BL (Backward-Left), or BR (Backward-Right).
      * @param drive   The TalonSRX motor controller for driving forward and backwards.
      * @param turn    The TalonSRX motor controller for turning the module into the correct orientation.
-     * @param gearRatio     The gear ratio for the <i> turn </i> motor controller.
+     * @param turnGearRatio     The gear ratio for the <i> turn </i> motor controller.
      *                      Used for determining the angle of the module.
      * @param driveModifier     A double which controls the speed passed into drive.setSpeed()
      * @param maxSpeed          The maximum speed, in m/s, of the SwerveModule. This should be the same speed when normalizing.
      * @param reversed          Whether or not to reverse the turn motor controller.
      * @param turnZero          The desired raw analog encoder value to reach during HomeAbsolute.
      * @param maxAnalog         The maximum value of the raw analog encoder.
      */
-    public SwerveModule(ModuleType type, WPI_TalonSRX drive, WPI_TalonSRX turn, double gearRatio, double driveModifier,
-                        double maxSpeed, boolean reversed, int turnZero, int maxAnalog) {
+    public SwerveModule(ModuleType type, WPI_TalonSRX drive, WPI_TalonSRX turn, double driveGearRatio, double wheelDiameter,
+                        double turnGearRatio, double driveModifier, double maxSpeed, boolean reversed, int turnZero, int maxAnalog) {
```

Comment:
Not critical, but this constructor is taking so many parameters that it might time to consider changing it to take a configuration object that is created via the [Immutables](https://immutables.github.io/) annotation processor.